.travis.yml

```diff
@@ -8,4 +8,8 @@ rvm:
   - 2.3.0
   - jruby
 
+script:
```

Comment:
Except the rspec tasks are inside a conditional. Don’t wanna mess with that right now.